src/components/flashcard/card-face/card-face.scss

```diff
@@ -18,6 +18,7 @@ $button-strip-height: 35px;
     width: 100%;
     display: flex;
     justify-content: space-between;
+    flex-direction: row-reverse;
```

Comment:
I am removing the speaker icon and button from the DOM when the text of the card is empty. When that happens, the kebab menu takes the place of the speaker because of the `justify-content: space-between;` property. 

We can flip the natural ordering of these icons, so have kebab on the left and speaker on the right if that would be better

Reply:
oh I see, that makes more sense; this is a good solution to the alternative which is an empty block element.
and no we shouldn't swap the ordering, kebab-menus should be on the right from a UX standpoint